content/en/docs/distributions/ibm/deploy/install-kubeflow-on-iks.md

```diff
@@ -279,15 +480,16 @@ to fill a URL for AppID to redirect to Kubeflow. The URL should look like `https
 2. Update the secret `appid-application-configuration` with the updated Kubeflow FQDN to replace `<kubeflow-FQDN>` in below command:
 
 ```SHELL
-redirect_url=$(printf https://<kubeflow-FQDN>/login/oidc | base64 -w0) \
- kubectl patch secret appid-application-configuration -n istio-system \
- -p $(printf '{"data":{"oidcRedirectUrl": "%s"}}' $redirect_url)
+export REDIRECT_URL=https://<kubeflow-FQDN>/login/oidc
+export PATCH=$(printf '{"data": {"REDIRECT_URL": "%s"}}' $REDIRECT_URL)
+
+kubectl patch configmap appid-application-configuration -n istio-system -p=$PATCH
 ```
 
 3. Restart the pod `authservice-0`:
 
 ```shell
-kubectl rollout restart statefulset authservice -n istio-system
+kubectl delete po authservice -n istio-system
```

Comment:
shall we keep `kubectl rollout restart statefulset authservice -n istio-system`?

Reply:
for some reason. rollout was not creating a new pod. the easiest way i found it picking up the cm and secret changes was to delete the pod. I can investigate more about rollout restart.